Cargo.toml

```diff
@@ -3,7 +3,7 @@
 resolver = "2"
 members = ["embedded-fatfs", "block-device-driver", "block-device-adapters", "sdspi"]
 # exclude embedded examples because cargo doesn't do well with multi-arch workspaces
-exclude = ["examples/esp32c6"]
+exclude = ["examples/esp32c6", "examples/rp2040"]
 
 [patch.crates-io]
-aligned = { git = "https://github.com/MabezDev/aligned", branch = "a1" }
\ No newline at end of file
+aligned = { git = "https://github.com/MabezDev/aligned", branch = "a1" }
```

Comment:
I just published aligned 0.4.2, so we can remove these patches now :)

Reply:
Great. Fixedup